Sources/BSON/BSONNulls.swift

```diff
@@ -70,6 +80,16 @@ internal struct BSONUndefined: BSONValue, Equatable {
         self = BSONUndefined()
     }
 
+    /// Converts this `BSONUndefined` to a corresponding `JSON` in relaxed extendedJSON format.
+    internal func toRelaxedExtendedJSON() -> JSON {
+        self.toCanonicalExtendedJSON()
+    }
+
+    /// Converts this `BSONUndefined` to a corresponding `JSON` in canonical extendedJSON format.
+    internal func toCanonicalExtendedJSON() -> JSON {
+        ["$undefined": .bool(true)]
```

Comment:
we can just use the boolean literal here `true`

Reply:
yep